### PR TITLE
Deduplicate queries retrieve builder view

### PR DESCRIPTION
**Pull request by brunobord**
refs #157

- [x] cherry-pick what comes from #158
- [x] fixup if necessary,
- [x] add changelog,
- [x] when reviewed, autosquash

### CODE REVIEW

File: demo/tests/perfs/tests_integration.perf.yml [hunk 1]
```diff
@@ -6,3 +6,12 @@ TestContextFormEndPoint.test_queryset:
 - db: SELECT ... FROM "formidable_item" WHERE "formidable_item"."field_id" IN (...) ORDER BY "formidable_item"."order" ASC
 - db: SELECT ... FROM "formidable_validation" WHERE "formidable_validation"."field_id" IN (...)
 - db: SELECT ... FROM "formidable_default" WHERE "formidable_default"."field_id" IN (...)
+UpdateFormTestCase.test_queryset_on_get:
+- db: 'SELECT ... FROM "formidable_formidable" WHERE "formidable_formidable"."id" = #'
+- db: SELECT ... FROM "formidable_field" WHERE "formidable_field"."form_id" IN (#) ORDER BY "formidable_field"."order" ASC
+- db: 'SELECT ... FROM "formidable_field" WHERE "formidable_field"."form_id" = # ORDER BY "formidable_field"."order" ASC'
```
Comment (moumoutte) [inline demo/tests/perfs/tests_integration.perf.yml]: Hum seems to have an extra queries here ?

Comment (moumoutte) [inline demo/tests/perfs/tests_integration.perf.yml]: Actually the line above.

Comment (brunobord) [inline demo/tests/perfs/tests_integration.perf.yml]: mmmmm... I don't know, I didn't try to change anything on your **code**, only the `.yml` files ; it's possible that your previous perf PRs had a side-effect on the queries generated in the next perf PRs.
Maybe you'd want to add commits to this branch and eventually tackle one last performance issue? or we may merge "as is", and we open an issue to resolve this one? your choice, this PR is the last one to integrate into django-formidable, it won't have any bad side-effect, IMHO

Comment (moumoutte) [inline demo/tests/perfs/tests_integration.perf.yml]: I think I miss this one , do you mind if I add a commit ?

**Pull request merged**